beast/observationmodel/ast/make_ast_input_list.py

```diff
@@ -365,3 +365,121 @@ def pick_models(
         ast_params.write(outfile_params, overwrite=True)
 
     return sedsMags
+
+
+def supplement_ast(
+    sedgrid_fname,
+    filters,
+    nAST=1000,
+    existingASTfile=None,
+    outASTfile=None,
+    outASTfile_params=None,
+    mag_cuts=None,
+):
+    """
+    Creates an additional fake star catalog from a BEAST model grid
+    that fulfills the customized conditions to supplement input ASTs.
+    If the existing input AST parameter file is given, already selected
+    models will be excluded from this process.
```

Comment:
This should state that the input ASTs are picked randomly from the remaining models.

Reply:
Done!